addons/web/static/src/scss/bootstrap_overridden_frontend.scss

```diff
@@ -18,6 +18,10 @@ $primary: $blue !default;
 // See https://www.w3.org/TR/WCAG20/#visual-audio-contrast-contrast
 $min-contrast-ratio: $o-frontend-min-contrast-ratio !default;
 
+// Color contrast variables used in the color-contrast function
+$color-contrast-dark: $black !default; // BS Default
+$color-contrast-light: $white !default; // BS Default
```

Comment:
I don't understand why we need to redefine this here since it's already done in the ` _variable.scss` file

Reply:
It's to be able to use the color-contrast function with the default value of `$color-contrast-light` and `$color-contrast-dark`, if we don't the SCSS won't compile because the `_variables.scss` file comes after the `web._assets_frontend_helpers`.
<img width="512" alt="image" src="https://github.com/user-attachments/assets/61494e3b-16ed-469b-8b5e-f94ccfc814e2" />